src/D2L.Bmx/Program.cs

```diff
@@ -1,3 +1,73 @@
+using System.CommandLine;
 
-// See https://aka.ms/new-console-template for more information
-Console.WriteLine( "Hello, World!" );
+var orgOption = new Option<string?>(
+	name: "--org",
+	description: "the okta org api to target" );
+var userOption = new Option<string?>(
+	name: "--user",
+	description: "the user to authenticate with" );
+var accountOption = new Option<string?>(
+	name: "--account",
+	description: "the account name to auth against" );
+var roleOption = new Option<string?>(
+	name: "--role",
+	description: "the desired role to assume" );
+var durationOption = new Option<int?>(
+	name: "--duration",
+	description: "duration of session in minutes",
+	getDefaultValue: () => 60 );
+var nomaskOption = new Option<bool>(
+	name: "--nomask",
+	description: "set to not mask the password, helps with debugging",
+	getDefaultValue: () => false );
+var printOutputOption = new Option<string?>(
+	name: "--output",
+	description: "the output format [bash|powershell]" );
+var writeOutputOption = new Option<string?>(
+	name: "--output",
+	description: "write to the specified file instead of ~/.aws/credentials" );
+var profileOption = new Option<string?>(
+	name: "--profile",
+	description: "aws profile name" );
+
+var rootCommand = new RootCommand();
+
+var printCommand = new Command( "print", "Print the long stuff to screen" )
+	{
+		accountOption,
+		durationOption,
+		nomaskOption,
+		orgOption,
+		printOutputOption,
+		roleOption,
+		userOption,
+	};
+
+printCommand.SetHandler(
+	Print.PrintHandler,
+	orgOption, userOption, accountOption, roleOption, durationOption, nomaskOption, printOutputOption
+ );
```

Comment:
it bothers me that we have a long list of options/handler arguments here, and most of them are of the same type `string?`. seems very prone to accidental mismatch.
I wonder if there's a better way. but this is fine for now.